packages/ui-components/src/components/relative-time-picker/relative-time-picker.config.ts

```diff
@@ -0,0 +1,429 @@
+import { ERelativeTimeComparisonConfig, IRelativeTimePickerOption } from './relative-time-picker.types';
+
+export const TIME_RANGE_UPDATE_INTERVAL = 10000;
+export const CUSTOMIZE_INTERVAL_KEY = 'customize-interval';
+export const CUSTOMIZE_INTERVAL_LABEL = 'Customize Interval';
+export const DROPDOWN_MAX_HEIGHT = '290px';
+
+export const TIMEZONES_PLACEHOLDER = 'Select a timezone';
+export const TIMEZONES_SEARCH_PLACEHOLDER = 'Search for a timezone';
+
+export const DEFAULT_RELATIVE_TIME_OPTIONS_GROUPS: IRelativeTimePickerOption[][] = [
+	[
+		{
```

Comment:
If we want to select a subset of these options, will we need to define the entire groups? 
Wouldn't it be better if this was indexed by some key and we could select which options we want by passing a set of keys?